pages/apim/3.x/kubernetes/apim-kubernetes-tools.adoc

```diff
@@ -0,0 +1,36 @@
+[[apim-kubernetes-tools]]
+= Gravitee Kubernetes Tools
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_tools.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+== Overview
+
+Gravitee provides two Kubernetes tools:
+
+  * The Gravitee Kubernetes Operator [GKO]
+  * The Gravitee Kubernetes Controller (plugin)
+
+== Gravitee Kubernetes Operator
+
+[label label-version]#New in version 3.19.0#
+
+APIM 3.19.0 has introduced the Gravitee Kubernetes Operator [GKO] - a new technical component designed to be deployed on a Kubernetes Cluster.
+
+You can use the GKO in the following ways:
+
+  * To define, deploy, and publish APIs to your API Portal and API Gateway and to manage Custom Resource Definitions (CRDs) as part of the process.
+  * To use the GKO as an Ingress Controller for deploying Ingresses to your API Gateway.
+  * To deploy Gravitee products such as API Management, Access Management, or Alert Engine.
+  * To improve automation processes through covering CICD aspects when using Kubernetes with APIM.
+  * To manage most API Management resources without directly relying on the Console or on the Management API.
```

Comment:
It's not available yet, it is our objective ;)

Reply:
So, just remove:

 * To manage most API Management resources without directly relying on the Console or on the Management API.

Or do we want to keep it as a "promise"?

Reply:
May be a kind of check box or some badging to flag features as:
  - alpha
  - released
  - upcoming

Reply:
Every thing being alpha for now :)

Reply:
We could only list the first point as existing functionality, and post everything else as "In the future" thing?

Reply:
Rephrased to:

You can use the GKO to define, deploy, and publish APIs to your API Portal and API Gateway and to manage Custom Resource Definitions (CRDs) as part of the process.

> In future releases, the GKO will support additional functionality to enable the following:
> 
>   * Using the GKO as an Ingress Controller for deploying Ingresses to an API Gateway.
>   * Deploying Gravitee products (API Management, Access Management, Alert Engine).
>   * Improving automation processes through covering CICD aspects when using Kubernetes with APIM.
>   * Managing most API Management resources without directly relying on the Console or on the Management API.

LMK if that is correct.